.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -20,6 +20,12 @@ See [DEVELOPMENT.md](https://github.com/aws-deadline/deadline-cloud/blob/mainlin
 - Are relevant docstrings in the code base updated?
 - Has the README.md been updated? If you modified CLI arguments, for instance.
 
+### Does this PR introduce new dependencies?
```

Comment:
Another option is to have two checkboxes:

*   [ ] This PR adds one or more new dependency Python packages. I acknowledge I have reviewed the considerations for adding dependencies in `DEVELOPMENT.md`.
*   [ ] This PR does not add any new dependencies.

Reply:
Adopted this checkbox idea, also changed the link to `DEVELOPMENT.md`